src/components/LHNOptionsList/OptionRowLHNData.js

```diff
@@ -88,6 +92,14 @@ function OptionRowLHNData({
     const parentReportAction = parentReportActions[fullReport.parentReportActionID];
 
     const optionItemRef = useRef();
+
+    const lastTransaction = useMemo(() => {
```

Comment:
I think this name is a bit misleading. If `sortedReportActions` doesn't have a linked transaction, we'd store an empty object here and not the last transaction

Reply:
how does `transactionOfLastReportAction` sound?

Reply:
we could just call it `linkedTransaction`

Reply:
I think we always only care about the last action in the LHN

Reply:
So I'll rename it to `linkedTranscation`. It is a required dependency because this specific data of scan status is held in the `transacation` and not in `reportAction`. So if `transaction` changes and it is not a depedency, LHN wouldnt update and be stale